inst/pages/add-comm-typing.Rmd

```diff
@@ -348,7 +348,7 @@ tse2 <- transformAssay(tse2, MARGIN = "rows", assay.type = "clr", method = "stan
 # get top taxa
 tse2 <- tse2[is.element(rownames(tse2), rel_taxa), ]
 
-mat <- assays(tse2)$z
```

Comment:
This does not work. assays() return a list, assay() fetch matrix. 

this should be assay(tse2, "standardize")

Reply:
hmm I'm not sure I get it.  Is this not the desirable change?
`mat <- assays(tse2, "standardize")`

Reply:
```
tse <- transformAssay(tse, method = "relabundance")

# Does not work
assays(tse, "relabundance")
# Does work
assays(tse)$relabundance
# Does work and preferred
assay(tse, "relabundance")
```

Reply:
Thanks